NEWS.md

```diff
@@ -188,6 +188,8 @@ External dependencies
 
 Tooling Improvements
 ---------------------
+* `Base.Experimental.enable_gc_logging(true)` can be used to log each garbage collection, with the
+  time it took and the amount of memory that was collected ([#43511]).
```

Comment:
FYI @vilterp: I pushed up a commit to add the reference to the PR, which is the expected format for this. :)